ext/EnzymeStaticArraysExt.jl

```diff
@@ -14,22 +14,26 @@ end
 
 @inline Enzyme.specialize_output(output, input::StaticArray) = convert(SArray, output)
 
-@inline function Enzyme.onehot(x::StaticArrays.SArray{S, T, N, L}) where {S, T, N, L}
-    ntuple(Val(L)) do i
+@inline function Enzyme.onehot(x::StaticArrays.SArray{S, T, N, L}; stacked::Union{Val{true}, Val{false}} = Val(false)) where {S, T, N, L}
```

Comment:
I'm also debating if we should make this a proper non kw argument (essentially because while the outer function is marked inline, I don't know a away to make the inner kwfunc inline [and we really do want that here]